climpred/tests/test_smoothing.py

```diff
@@ -133,3 +99,108 @@ def test_compute_after_smooth_goddard_2013(
     actual = compute_perfect_model(PM_da_initialized_3d_full, PM_da_control_3d_full)
     north_atlantic = actual.sel(lat=slice(40, 50), lon=slice(-30, -20))
     assert not north_atlantic.isnull().any()
+
+
+@pytest.mark.parametrize('tsmooth_kws', [{'lead': 2}, {'lead': 4}, {'time': 4}])
+def test_HindcastEnsemble_temproal_smooth_leadrange(hindcast_recon_3d, tsmooth_kws):

```

Comment:
Fix name to `test_HindcastEnsemble_temporal_smooth_leadrange`. Add description:

`"""Tests that `HindcastEnsemble` can verify with temporal smoothing over various lead ranges."""`